curvefs/conf/client.conf

```diff
@@ -15,7 +16,7 @@ metaCacheOpt.metacacheGetLeaderRPCTimeOutMS=1000
 
 #excutorOpt
 excutorOpt.maxRetry=1000000
-excutorOpt.retryIntervalUS=200
+excutorOpt.retryIntervalUS=500
```

Comment:
//metaCacheOpt
// Gets the number of retries for the leader
metaCacheOpt.metacacheGetLeaderRetry=3
// Need to sleep for a period of time before each get leader retry
metaCacheOpt.metacacheRPCRetryIntervalUS=100000
// RPC timeout of get leader
metaCacheOpt.metacacheGetLeaderRPCTimeOutMS=1000

// excutorOpt rpc with metaserver
// rpc retry times with metaserver
excutorOpt.maxRetry=1000000
// Retry sleep time between failed RPCs
excutorOpt.retryIntervalUS=500
// RPC timeout for communicating with metaserver
excutorOpt.rpcTimeoutMS=1000
// The maximum timeout RPC time of the retry request. 
// The timeout time will follow the exponential backoff policy.
// Because the timeout occurs when the network is congested, the RPC timeout needs to be increased
excutorOpt.maxRPCTimeoutMS=8000
// Maximum sleep time between retry requests.
// when the network is congested or the metaserver is overloaded,
// it is necessary to increase the sleep time
excutorOpt.maxRetrySleepIntervalUS=8000000

    
        
          
    

        
    
    @@ -40,10 +41,11 @@ rpc.defer.close.second=600
  
excutorOpt.minRetryTimesForceTimeoutBackoff=5
excutorOpt.maxRetryTimesBeforeConsiderSuspend=20